pkg/runner/runner.go

```diff
@@ -86,7 +88,8 @@ func (r *Runner) create(options *flags.Options) (err error) {
 		return &ErrVersion{siVersion, cloudVersion}
 	}
 	var path string
-	path, err = r.imgDriver.Create(options.Release, options.Channel, options.Arch, siVersion)
+	dotRelease := addDots(options.Release)
```

Comment:
I would add a comment here saying that the dot is only added if the release is a number of 4 digits, like 15.04